ci-operator/step-registry/sandboxed-containers-operator/pre/sandboxed-containers-operator-pre-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: sandboxed-containers-operator-pre
+  steps:
+    - ref: sandboxed-containers-operator-get-kata-rpm
+    - ref: sandboxed-containers-operator-peerpods-param-cm
```

Comment:
This creates peer-pods-cm? If peer-pods-cm is in the cluster, peerpods-param-cm is used to create it because we used to create peer-pods-secrets and we can't pass that in Jenkins.

It is much better if the CI creates peer-pods-cm directly and we remove it from the test automation.

Reply:
Currently that step creates the `peerpods-param-cm` configmap (see https://github.com/openshift/release/blob/master/ci-operator/step-registry/sandboxed-containers-operator/peerpods/param-cm/sandboxed-containers-operator-peerpods-param-cm-commands.sh#L106)

It also creates `peerpods-param-secrets` (https://github.com/openshift/release/blob/master/ci-operator/step-registry/sandboxed-containers-operator/peerpods/param-cm/sandboxed-containers-operator-peerpods-param-cm-commands.sh#L66)

Does it work fine with current test automation?

And yes, we can move some of the logic from test automation to prow steps.

Reply:
I've only tested with kata, not peer-pods